lib/tapjoy/ldap/api/user.rb

```diff
@@ -0,0 +1,112 @@
+require 'memoist'
+module Tapjoy
+  module LDAP
+    module API
+      module User
+        class << self
+          extend Memoist
+          def create(fname, lname, type, group)
+            # Properly capitalize names
+            fname, lname = [fname, lname].map(&:titleize)
```

Comment:
For title case strings, wouldn't the method be less pleasant than just including active support?

I'm thinking it would look something like this:
```ruby
def method(array_of_strings)
  array_of_strings.each do |i|
    i = "#{i[0]}#{i[1..i.size]}"
  end

  array_of_strings.join(' ')
end```
    
It's not clear to me that this is something we'd prefer.